presto-docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -89,6 +89,18 @@ Finally, you can access the ``clicks`` table in the ``web`` schema::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``postgresql`` in the above examples.
 
+Push down
```

Comment:
We use the term "pushdown" 59 places, and "push down"  in only 13 places. Words generally become combined at some point, and I think "pushdown" is already there. Consider e-mail vs email.

Example: https://docs.microsoft.com/en-us/sql/relational-databases/polybase/polybase-pushdown-computation

Existing usage in session properties:

* `predicate_pushdown_use_table_properties`
* `s3_select_pushdown_enabled`
* `projection_pushdown_enabled`

Reply:
So should I change it across it all?

Reply:
As a separate commit, yes